### PR TITLE
Apply access list url normalization

### PR DESCRIPTION
**Pull request by yaron2**
This PR fixes a loophole in access list routes by normalizing URLs and accounting for HTTP case sensitive routing.

A new package dependency is used for normalization.

### CODE REVIEW

File: pkg/grpc/api.go [hunk 1]
```diff
@@ -163,6 +164,14 @@ func (a *api) CallLocal(ctx context.Context, in *internalv1pb.InternalInvokeRequ
 	return resp.Proto(), err
 }
 
+func normalizeOperation(operation string) string {
+	s, err := purell.NormalizeURLString(operation, purell.FlagsUsuallySafeGreedy|purell.FlagRemoveDuplicateSlashes)
+	if err != nil {
+		panic(err)
```
Comment (mukundansundar) [inline pkg/grpc/api.go:170]: i think this should be an error returned. 
In line 188, surround by if clause and return err message to caller.

Comment (yaron2) [reply pkg/grpc/api.go:170]: oops, yes. that should be a return

**Review by artursouza — APPROVED** [review]

**Comment by 1046102779** [comment]
#3142 issue: use normalizeOperation necessity

**Pull request merged**